.github/workflows/Build.yml

```diff
@@ -484,281 +490,277 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./ci
 
-  #43
   verible-linux:
-    runs-on: "ubuntu-20.04"
+    runs-on: [self-hosted, Linux, X64]
+    container: ubuntu:20.04
     env:
-      PACKAGE: "sv-front/verible"
-      USE_SYSTEM_GCC_VERSION: "9"
+      PACKAGE: "misc/magic"
```

Comment:
`sv-front/verible`

Reply:
Fixed. Just in case I grepped through the file to check if there's no similar mistake.